scripts/solutionsLib.py

```diff
@@ -1064,11 +1064,11 @@ def executeCommand(self, com, index=0):
                     self.getProcessInfoValue(processKey, 'max_cached_scale', index))
                 if os.path.isfile(tileSchemeMtc):
                     cachepath = os.path.join(self.getProcessInfoValue(processKey, 'in_cache_location', index), self.getProcessInfoValue(processKey, 'in_cache_name', index))
-                    lodNodesList = returnLevelDetails(os.path.join(cachepath, 'conf.xml'))
+                    lodNodesList = self.returnLevelDetails(os.path.join(cachepath, 'conf.xml'))
                     lodNodesList = sorted(lodNodesList, key=lambda k: int(k['level']))
                     maxLODNode = lodNodesList[-1]
                     maxScale = maxLODNode['scale']
-                    modifyConfProperties(os.path.join(cachepath, 'conf.properties'), maxScale)
+                    self.modifyConfProperties(os.path.join(cachepath, 'conf.properties'), maxScale)
```

Comment:
@bikesbade  Put the whole block in a try:  exception block to trap any errors in (returnLevelDetails, modifyConfProperties) functions. I see the XMLs loaded & modified without any error checks within the functions. Please return False if an exception is thrown with an appropriate error message logged to signal w/f failures. Thanks!